image-segmentation/custom-recipes/score_image/recipe.py

```diff
@@ -0,0 +1,140 @@
+#############################
+# Your original recipe
+#############################
+
+# -------------------------------------------------------------------------------- NOTEBOOK-CELL: CODE
+# -*- coding: utf-8 -*-
+import dataiku
+import pandas as pd, numpy as np
+from dataiku import pandasutils as pdu
+from dataiku.customrecipe import *
+import os
+import sys
+import random
+import math
+import numpy as np
+import skimage.io
+import matplotlib
+import matplotlib.pyplot as plt
+import itertools
+import colorsys
+from skimage.measure import find_contours
+from matplotlib import patches,  lines
+from matplotlib.patches import Polygon
+
+# Checking whether pycocotools is installed
+try :
+    import pycocotools
+except : 
+    print("##### WARNING ##### Couldn't find pycocotools in installed packages : will proceed to installing the package in plugin codenv")
+    client = dataiku.api_client()
+    codenv = client.get_code_env('python','plugin_image-segmentation_managed')
+    codenv_def = condev.get_definition()
+    codenv_def['specPackageList'] = 'git+https://github.com/waleedka/coco.git#subdirectory=PythonAPI'
+    codenv.set_definition(codenv_def)
+    
+    print('#'*50)
+    print('Start updating codenv plugin_image-segmentation_managed with git+https://github.com/waleedka/coco.git#subdirectory=PythonAPI')
+    codenv.update_packages()
+    print ('Codenv updated')
+# Import Mask RCNN
+from mrcnn import utils
```

Comment:
multiple line import of the same package, should merge them

Reply:
ok done